__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const testData = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("GET /api/categories", () => {
+  test("status: 200, responds with an array of category objects containing slug and description properties", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        const { categories } = body;
+        expect(categories).toBeInstanceOf(Array);
+
+        categories.forEach((category) => {
```

Comment:
We need to also check the length of the arrray. A test will only fail if an expect statement fails, so that means if the array is length 0, then the forEach will never run and will never fail. We need to make sure that the array is the correct length as well as checking the contents of it